src/celeritas/optical/ModelImporter.hh

```diff
@@ -0,0 +1,122 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/optical/ModelImporter.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include "celeritas/io/ImportOpticalModel.hh"
+
+namespace celeritas
+{
+class MaterialParams;
+struct ImportData;
+struct ImportOpticalRayleigh;
+struct ImportWavelengthShift;
+
+namespace optical
+{
+struct ModelBuilder;
+class MaterialParams;
+class ImportedModels;
+class ImportedMaterials;
+//---------------------------------------------------------------------------//
+/*!
+ * Construct Celeritas optical model builders from imported data.
+ */
+class ModelImporter
+{
+  public:
+    //!@{
+    //! \name Type aliases
+    using IMC = ImportModelClass;
```

Comment:
Gosh until I looked this out to figure where it was coming from I had no idea there was an `optical::ImportModelClass`.